### DISCUSSION TITLE
Roadblocks and cartpushers

### DISCUSSION THREAD

**Discussion by mathmaj** [Keriew/augustus#310] [General]
I'm sure you've already thought about it, But would it be possible to block the cartpusher with the roadblock?
This would keep them in a defined area, so that they do not go to the other end of the city to carry goods.

**Comment by crudelios** [comment]
I'm afraid this is not possible. 

Roadblocks only affect random walkers, not walkers who have a specific destination.

As cartpushers always have a destination, there isn't an easy way to change the behaviour to consider roadblocks when determining the path.

**Reply by mathmaj** [reply]
I understand. Thanks you for your answer!
